app/graphql/types/partner_type.rb

```diff
@@ -3,18 +3,38 @@ class PartnerType < Types::BaseObject
 
     description 'A Partner who runs Events'
 
+    # placecalID
     field :id, ID, null: false
     field :name, String, null: false
-    field :description, String
     field :summary, String
+    field :description, String
+
+    field :accessibility_summary, String, method: :accessibility_info
+
+    field :logo, String, method: :image
+    field :address, AddressType
 
     field :url, String
```

Comment:
Just to confirm theres not better types to use than string for email, phone, url etc?

Reply:
graphql has very limited basic types: https://graphql.org/graphql-js/basic-types/